flowforge/materials/Fluid.py

```diff
@@ -176,14 +215,15 @@ class Hitec(Fluid):
     Molar Percent: 7-49-44 mol%
     Weight Percent: 7-40-53 wt.%
 
-    Caveates:
-    In this section a list of temperature dependent caveates are presented
+    Caveats:
+    In this section a list of temperature dependent caveats are presented
     with uncertainties from the liturature
 
-    Thermal conductivity:       valid for temp range 373-773 K with plus minus 5% uncertainty  [1]
-    Density:                    valid for temp range 470-870 K with plus minus 2% uncertainty  [2]
-    Dynamic viscosity:          valid for temp range 420-710 K with plus minus 16% uncertainty [2]
-    Specific heat:              valid for temp range 426-776 K with plus minus 5% uncertainty  [2]
+    Thermal conductivity:       valid for temp range 373-773 K with ± 5% uncertainty  [1]
+    Density:                    valid for temp range 470-870 K with ± 2% uncertainty  [2]
+    Dynamic viscosity:          valid for temp range 420-710 K with ± 16% uncertainty [2]
+    Surface tension:            valid for temp range 570-670 K with ± 10% uncertainty [2]
+    Specific heat:              valid for temp range 426-776 K with ± 5% uncertainty  [2]
 
     References
```

Comment:
For some reason it won't let me comment on the line I want to, but this comment refers to the HiTech `temperature` method.  Where did that come from?  Surely there's some better way to do it, what ever it is.

Reply:
I greatly agree. It came from the non-linearity of the heat capacity and documented on the SyTH side. But that might have to be another issue that is independent from added surface tension issue. What do you think?

Reply:
ok, you'll need to explain the math to me.  Also, these two assertions:

```
assert np.sum(np.isclose(T_roots.imag, 0) & (T_roots.real >= 0)) == 1
```

Why did you do the `np.sum(...) == 1`?  It seems unnecessary

Reply:
root find derivation [here](https://github.com/UT-Computational-NE/syth/issues/127#issuecomment-2152849133)

Reply:
Need to make update on commenting on the root find for temp of Hitec.

Reply:
Looks good.